### PR TITLE
Updating GSoC 2022 projects with defaults and adding expected outcomes

### PR DESCRIPTION
**Pull request by nate-double-u**
Updating default level of effort and difficulty. Adding Expected Outcomes. Updating descriptions where needed.

These do not have to be the final values, but I want every project to have a complete entry.

### CODE REVIEW

File: summerofcode/2022.md [hunk 1]
```diff
@@ -160,21 +160,21 @@ The idea is to create a component that is capable of translating OCI format to V
...
-- Difficulty: Easy, Medium, or Hard
+- Expected project size: 175 Hours
+- Difficulty: Medium
 - Upstream Issue (URL): https://github.com/oam-dev/kubevela/issues/3205
 
 #### Enhance multi-cluster observability
 
 - Description: Extend the observability of multi-cluster information on KubeVela control plane
-- Expected outcome:
+- Expected outcome: Provide extensible cluster infromation, starting with providing more information such as: the health status, available cpu cores, whether it is a GPU cluster, the stability of connections with KubeVela control plane, etc.
```
Comment (amye) [inline summerofcode/2022.md:173]: infromation

**Review by amye — APPROVED** [review]
One typo, not a big deal

File: summerofcode/2022.md (hunk 1 above)
Comment (nate-double-u) [reply summerofcode/2022.md:173]: good catch, thx

**Comment by nate-double-u** [comment]
Thanks, typo corrected.

**Pull request merged**